src/GFDL-1.1.xml

```diff
@@ -3,13 +3,11 @@
   <license licenseId="GFDL-1.1" isOsiApproved="false"
   name="GNU Free Documentation License v1.1"
   isDeprecated="true" deprecatedVersion="3.0">
+    <obsoletedBy>GFDL-1.1-only</obsoletedBy>
+    <obsoletedBy expression="GFDL-1.1+">GFDL-1.1-or-later</obsoletedBy>
```

Comment:
The or-later-ness of this identifier is not clear to me, because (as of list 3.0) the grant for `GFDL-1.1` includes “[or any later version][1]”.  Because there is no `ONLY` operator, I'm guessing that `GFDL-1.1` means `GFDL-1.1-only` and that you'd need `GFDL-1.1+` (pre-3.0) to mean `GFDL-1.1-or-later`, despite the wording we claimed in `<standardLicenseHeader>`.  But we may want to update the standard license header and/or file an erratum (#598) for clarity going forward.

This issue also affects version 1.2 and 1.3 of the GFDL.

[1]: https://github.com/spdx/license-list-XML/blob/v3.0/src/GFDL-1.1.xml#L17